packages/components/src/components/post-popovercontainer/post-popovercontainer.scss

```diff
@@ -11,7 +11,11 @@
 // Can be removed as soon as popover is supported by all major browsers
 // https://caniuse.com/?search=popover
 @layer polyfill {
-  @include meta.load-css('@oddbird/popover-polyfill/dist/popover.css');
+  @supports not selector(:popover-open) {
+    [popover]:not(.\:popover-open) {
+      display: none;
+    }
+  }
```

Comment:
Why removing this polyfill ? Why in this PR ? Maybe this is wrong as the package is still installed ?